### PR TITLE
fix: secure icons in lock screen are too small

### PR DESCRIPTION
**Pull request by zsien**
Issues: linuxdeepin/developer-center#8504

### CODE REVIEW

File: common-plugin/networkdialog/networkpanel.cpp [hunk 1]
```diff
@@ -395,7 +395,7 @@ void NetworkPanel::updateItems()
                 // 连接隐藏网络
                 WirelessItem *apCtrl = findWirelessItem(nullptr, device);
                 if (!apCtrl) {
-                    apCtrl = new WirelessItem(m_netListView->viewport(), device, nullptr, this);
+                    apCtrl = new WirelessItem(m_netListView, device, nullptr, this);
```
Comment (kegechen) [inline common-plugin/networkdialog/networkpanel.cpp:398]: 这里是改的啥

Comment (zsien) [reply common-plugin/networkdialog/networkpanel.cpp:398]: parent

Comment (kegechen) [reply common-plugin/networkdialog/networkpanel.cpp:398]: listview的item的parent应该是viewport， 因为绘制在viewport上的

Comment (zsien) [reply common-plugin/networkdialog/networkpanel.cpp]: done

**Review by 18202781743 — APPROVED** [review]

**Review by kegechen — APPROVED** [review]

**Pull request merged**